docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+|Parity| The issue describes a gap in parity between two or more platforms.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Parity` |
+|Technical debt| The issue involves refactoring existing code to make it easier to maintain, follow best practices, improved test coverage, etc.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Technical debt` |
+|Team agility| An issue targeted to reduce friction to releasing new versions of the SDKs.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Team agility` |
+
+### Size
+
+Use these to assign an estimated level of effort to resolve an issue and assist with the estimation process.
+
+Color: All the labels in this category should use ![#91e3ea](https://via.placeholder.com/15/91e3ea/000000?text=+) `#91e3ea`.
+
+|Name| Description  | Example
+|---|---|------|
+|Small| The issue is simple and well understood, it will take a day or less to complete | `Size:S` |
+|Medium| The issue is not very complex and it is well understood, it will take 1 to 3 days to complete | `Size:M` |
+|Large| The issue complex but it is well understood, it will take 4 to 8 days to complete | `Size:L` |
+|Extra Large| The issue very complex or not very well defined, it will take 9 to 14 days to complete. In this case, it is probably better to rethink the issue and break it down in smaller tasks | `Size:XL` |
+
+### Automation
+
+These labels are applied to PRs and used to trigger or disable github workflows.
+
+Color: All the labels in this category should use ![#cccccc](https://via.placeholder.com/15/cccccc/000000?text=+) `#cccccc`.
+
+|Name| Description  | Example
+|---|---|------|
+|no parity| PR does not need to be applied to other languages. **Note:** if you don't apply the `No parity` to a dotnet PR, the automation workflow will generate parity issues in Python, JS and Java.  | `Automation:No parity` |
+|parity with dotnet| The PR needs to be ported to dotnet. | `Automation:Parity with dotnet` |
+|parity with JS| The PR needs to be ported to JS. | `Automation:Parity with JS` |
+|parity with Python| The PR needs to be  ported to Python. | `Automation:Parity with JS` |
+|parity with Java| The PR needs to be ported to Java. | `Automation:Parity with Java` |
+
+### DRI
+
+DRI labels are used for reporting outside the bot framework repositories and their names don't follow the standards described above. They should be applied based on the current DRI guide.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|Bot Services| Required for internal Azure reporting. Do not delete. Do not change color.| ![#e99695](https://via.placeholder.com/15/e99695/000000?text=+) `#e99695` | `Bot Services` |
```

Comment:
Can we be more elaborate on these labels? Things I am looking at:

- Do we always add "Bot Services" for every single bug? If not, what bugs should be exempted?
- Who do we consider "customer"? We have few types of "customer"
   - Inner ring (devs who own the project, definitely not customer)
   - SDK team not in the inner ring (e.g. Composer devs filling an issue to SDK)
   - Services team (e.g. filing a bug on Web Chat)
   - Partners inside Microsoft
   - Everyone else

Reply:
These labels are imposed by the DRI process, I don't think we can change them but I will ask.